docs/getting_started_on_linux.md

```diff
@@ -68,6 +68,42 @@ environment variable is set when you are building. You may find it useful to add
 `source [PATH TO VULKAN SDK]/setup-env.sh` to your `~/.bashrc` file to simplify
 environment variable setup.
 
+## Optional: Configure Git
+
+### Git SSH
+
+*   Generate SSH Key: `ssh-keygen -t rsa -b 4096 -C "EMAIL@email.com"`
+*   Add `~/.ssh.id_rsa.pub` key to GitHub
+*   Try a test connection `ssh git@github.com`
+
+### Other git config options
+
+```shell
+git config --global user.name "MY NAME"
+git config --global user.email "MY EMAIL"
+```
+
+### Clone
```

Comment:
Let's put this one level up. Cloning isn't part of configuring git.

```suggestion
## Clone
```

Reply:
Ok. Will change it